packages/react/src/components/Footer/__data__/language-items.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
@jeffchew Do we need to include full cc-lc for these IDs?

Reply:
no, these are only targeting language